geocity/apps/submissions/management/commands/fixturize.py

```diff
@@ -116,7 +116,7 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         self.stdout.write("Resetting database...")
         reset_db()
-        setup_media(images_folder)
+        # setup_media(images_folder)
```

Comment:
add `images_folder = None` in `generic_example.py`. 
This kind of edit can lose us some time, if we don't find why agenda fixturize don't work anymore